ocaml/xapi/xapi_host.ml

```diff
@@ -3127,3 +3127,22 @@ let emergency_clear_mandatory_guidance ~__context =
          info "%s: %s is cleared" __FUNCTION__ s
      ) ;
   Db.Host.set_pending_guidances ~__context ~self ~value:[]
+
+let start_ssh ~__context ~self =
+  try
+    Xapi_systemctl.start ~wait_until_success:false "sshd" ;
+    Xapi_systemctl.enable ~wait_until_success:false "sshd"
```

Comment:
This function want to enable ssh?  then `start_ssh` is not a good name, othewise, you do not need to enable ssh for every start.

BTW, is it better to `start` after `enble`?